src/hooks/customer/useIsPremiumUser.tsx

```diff
@@ -0,0 +1,21 @@
+import { gql } from '@apollo/client'
+
+import { useGetCurrentUserPremiumQuery } from '~/generated/graphql'
+
+gql`
+  query getCurrentUserPremium {
+    currentUser {
+      id
+      premium
+    }
+  }
+`
+
+export const useIsPremiumUser = () => {
```

Comment:
This is a bit too specific IMO, we should have a hook that allow to access the currentUser

Reply:
As discussed on Slack, we'll keep it like this until a bigger refactor planned on both orga and customer data management